src/CLI/actioner/sonic-cli-mac.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+###########################################################################
+#
+# Copyright 2019 Dell, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+###########################################################################
+
+## TODO: Update with network instance l2 yang
+
+import sys
+import time
+import json
+import ast
+import openconfig_interfaces_client #update with network instance client
+from rpipe_utils import pipestr
+from openconfig_interfaces_client.rest import ApiException 
+from scripts.render_cli import show_cli_output
+
+import urllib3
+urllib3.disable_warnings()
+
+
+plugins = dict()
+
+def register(func):
+    """Register sdk client method as a plug-in"""
+    plugins[func.__name__] = func
+    return func
+
+
+def call_method(name, args):
+    method = plugins[name]
+    return method(args)
+
+# Update with network instance API
+def generate_body(func, args):
+    body = None
+    if func.__name__ == 'patch_openconfig_interfaces_interfaces_interface_config_description':
+       keypath = [ args[0] ]
+       body = { "openconfig-interfaces:description": args[1] }
+    elif func.__name__ == 'patch_openconfig_interfaces_interfaces_interface_config_enabled':
+       keypath = [ args[0] ]
+       if args[1] == "True":
+           body = { "openconfig-interfaces:enabled": True }
+       else:
+           body = { "openconfig-interfaces:enabled": False }
+    else:
+       body = {}
+
+    return keypath,body
+
+def getId(item):
```

Comment:
This function is not needed, as this sorts the interface name. Please take this off. Need to move this function to a new file, sonic_cli_utils.py.

Reply:
Removed code from the file.